yarn.lock

```diff
@@ -3442,11 +3442,27 @@
     query-string "^7.0.0"
     react-is "^16.13.0"
 
+"@react-navigation/core@^6.2.2":
```

Comment:
This still looks strange, we have now two versions of core in yarn lock can you verify why. https://classic.yarnpkg.com/en/docs/cli/why

Reply:
Removed https://github.com/trezor/trezor-suite/pull/5751/commits/4bb811986b6e11032801424f316c647fba27656e